lib/plugins.js

```diff
@@ -251,15 +261,27 @@ function plugins (settings, deviceId) {
           cassettes,
           rates: buildRates(tickers),
           balances: buildBalances(balances),
-          coins: _.zipWith(_.assign, coinsWithoutRate, tickers),
+          coins: _.zipWith(_.assign, _.zipWith(_.assign, coinsWithoutRate, tickers), batchableCoins),
           configVersion,
           areThereAvailablePromoCodes
         }
       })
   }
 
   function sendCoins (tx) {
-    return wallet.sendCoins(settings, tx)
+    return wallet.supportsBatching(settings, tx.cryptoCode)
```

Comment:
supportsBatching doesn't need this accounts settings

Reply:
also, you're still using a function for each coin.
```suggestion
    return wallet.checkCryptoCode(tx.cryptoCode) && wallet.SUPPORTS_BATCHING
```
but if you prefer the way it is now, just leave it, it's ok :)

Reply:
Hmm, the generic wallet `supportsBatching` method needs the `settings` parameter, as it calls `fetchWallet(settings, cryptoCode)` before calling the specific wallet `supportsBatching` function. This last one might not need the `settings` parameter, at least at this point in time, I agree, as only bitcoind is having batching enabled. Once 3rd party services have batching, they might require the `settings` parameter.

About the 2nd comment, I decided to keep the individual wallet's `supportsBatching` as a function, despite having the `SUPPORTS_BATCHING` constant, as the function still needs to confirm the cryptoCode with `checkCryptoCode`, as all functions from the individual wallets have. The end result will still be the `SUPPORTS_BATCHING` constant, it's just that it need to confirm the cryptocode first. BTW, I don't think the suggestion on the comment makes sense, as `checkCryptoCode` is a promise and making `Promise && Boolean` does nothing I believe, could you please elaborate?

Reply:
You're right the suggestion doesn't work. Given a second thought it's more readable as it is.